cogs/deadline.py

```diff
@@ -251,6 +268,10 @@ async def changeduedate_error(self, ctx, error):
     @commands.command(name="duethisweek", pass_context=True,
                       help="check all the homeworks that are due this week $duethisweek")
     async def duethisweek(self, ctx):
+        if not ctx.channel.name == 'assignments':
+            await ctx.author.send('Please send questions to the #assignments channel.')
```

Comment:
'Please send questions to the #assignments channel.'

Should be:

'Please use this command inside the #assignments channel.'